lib/driverProviders/driverProvider.ts

```diff
@@ -56,7 +56,13 @@ export abstract class DriverProvider {
     if (this.config_.disableEnvironmentOverrides === true) {
       builder.disableEnvironmentOverrides();
     }
-    let newDriver = builder.build() as WebDriver;
+    let newDriver: WebDriver;
+    try {
+      newDriver = await builder.build();
+    } catch (e) {
+      e.code = 135;
```

Comment:
135?

Reply:
Probably should actually throw the right error instead of adding the correct error code.